tests/python/test_schedule_ops.py

```diff
@@ -1043,11 +1084,17 @@ def definition(self):
                 bias_shape = fd.define_vector([S0, S1, S2, S3], dtype=DataType.Int)
 
                 tv1 = fd.ops.abs(x)
-                x_reshape = fd.ops.reshape(tv1, new_shape=bias_shape)
-                y = fd.ops.add(x_reshape, bias)
+                self.x_reshape = fd.ops.reshape(tv1, new_shape=bias_shape)
+                y = fd.ops.add(self.x_reshape, bias)
                 fd.add_output(y)
 
             def schedule(self):
+                assert len(fd.sched.tensors()) == 5
+                assert (
+                    fd.sched.to_string(self.x_reshape)
+                    == "T5_l[ iS27{2}rf, iS28{( ceilDiv(( i0 * i1 ), 2) )}rf, iS30{3}rf, iS31{( ceilDiv(i2, 3) )}rf ]"
```

Comment:
I'm wondering how reliable this check is...

Reply:
I still have concern here on this check. exact string match seems way to flaky. But I also don't have a better idea to test this...

Reply:
We don't have to check for an exact string match. I just want to ensure we don't get a segmentation fault when accessing a concretized symbolic tensor.